src/protocol.py

```diff
@@ -445,7 +445,7 @@ def decryptAndCheckPubkeyPayload(data, address):
             return 'failed'
         try:
             decryptedData = cryptorObject.decrypt(encryptedData)
-        except:
+        except:  # noqa:E722
```

Comment:
Write a "FIXME" that we need to have a test for this.

Reply:
```
  # FIXME: use a proper exception after `pyelliptic.ecc` is refactored.
```

Reply:
Updated